HousingFinanceInterimApi.Tests/V1/TestHelpers/TestDataGenerator.cs

```diff
@@ -1,28 +1,47 @@
 using System;
 using AutoFixture;
+using Bogus;
 
-namespace HousingFinanceInterimApi.Tests.V1.TestHelpers
+namespace HousingFinanceInterimApi.Tests.V1.TestHelpers;
+
+
+public static class TestDataGenerator
 {
-    public static class TestDataGenerator
+    private static readonly Fixture _fixture = new();
+    private static readonly Faker _faker = new(locale: "en_GB");
+
+    public static string PropRef
     {
-        private static readonly Fixture _fixture = new();
-        private static readonly Bogus.Faker _faker = new();
-
-        /// <summary>
-        /// Property Reference known locally as  prop_ref
-        /// </summary>
-        /// <returns></returns>
-        public static string PropRef
+        get
         {
-            get
-            {
-                return string.Concat("9", _fixture.Create<int>().ToString().PadLeft(7, '0').AsSpan(0, 7));
-            }
+            return string.Concat("9", _fixture.Create<int>().ToString().PadLeft(7, '0').AsSpan(0, 7));
         }
+    }
 
-        public static string RentAccount()
-        {
-            return _faker.Random.Long(0, 9_999_999_999).ToString().PadLeft(10, '0');
-        }
+    public static string RentAccount()
+    {
+        return _faker.Random.Long(0, 9_999_999_999).ToString().PadLeft(10, '0');
     }
+
+}
+
+
+public static class CashDumpTestData
```

Comment:
I was unsure where to put this - here seems fine for now but it's a very specific kind of data and this could get out of hand if stuff like this is needed a lot in future. I'm thinking this is something to reconsider down the line though